.buildkite/pipeline.yml

```diff
@@ -164,9 +164,14 @@ steps:
       queue: "bigcores"
     commands:
       - set -x
-      - cd js
+      - cd go
+      - go mod vendor
```

Comment:
a small detail, but this can be a makefile dependency in `js/`

it's not a problem to have this step in buildkite, but it can be a problem to support people that didn't run this command

Reply:
it's already the case locally when you run `make gen` in `js/`

I call the commands directly in the pipeline to avoid runing docker in docker